test/acceptance/features/workloadResourceMapping.feature

```diff
@@ -779,3 +782,304 @@ Feature: Bind services to workloads based on workload resource mapping
+                  - id: $scenario_id-id
+                initContainerSpecs:
+                  - id: $scenario_id-init
+            """
+        And The Workload Resource Mapping is present
+            """
+            apiVersion: servicebinding.io/v1beta1
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: notpodspecs.stable.example.com
+            spec:
+                versions:
+                  - version: "*"
+                    volumes: .spec.volumeData
+                    containers:
+                      - path: .spec.containerSpecs[*]
+                        name: .id
+                        env: .envData
+                        volumeMounts: .volumeEntries
+            """
+        And Service Binding is applied
+            """
+            apiVersion: servicebinding.io/v1beta1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id-binding
+            spec:
+                service:
+                    name: $scenario_id-secret
+                    kind: Secret
+                    apiVersion: v1
+                workload:
+                    apiVersion: stable.example.com/v1
+                    kind: NotPodSpec
+                    name: $scenario_id-npc
+            """
+        And Service Binding is ready
+        And jsonpath "{.spec.containerSpecs[0].volumeEntries}" on "notpodspecs/$scenario_id-npc" should return "[{"mountPath":"/bindings/$scenario_id-binding","name":"$scenario_id-binding"}]"
+        And jsonpath "{.spec.initContainerSpecs[0].volumeEntries}" on "notpodspecs/$scenario_id-npc" should return no value
+        When The Workload Resource Mapping is present
+            """
+            apiVersion: servicebinding.io/v1beta1
+            kind: ClusterWorkloadResourceMapping
+            metadata:
+                name: notpodspecs.stable.example.com
+            spec:
+                versions:
+                  - version: "*"
+                    volumes: .spec.volumeData
+                    containers:
+                      - path: .spec.containerSpecs[*]
+                        name: .id
+                        env: .envData
+                        volumeMounts: .volumeEntries
+                      - path: .spec.initContainerSpecs[*]
+                        name: .id
+                        env: .envData
+                        volumeMounts: .volumeEntries
+            """
+        Then Service Binding is ready
+        And jsonpath "{.spec.containerSpecs[0].volumeEntries}" on "notpodspecs/$scenario_id-npc" should return "[{"mountPath":"/bindings/$scenario_id-binding","name":"$scenario_id-binding"}]"
+        And jsonpath "{.spec.initContainerSpecs[0].volumeEntries}" on "notpodspecs/$scenario_id-npc" should return "[{"mountPath":"/bindings/$scenario_id-binding","name":"$scenario_id-binding"}]"
+
+    @spec
+    Scenario: Altering a workload mapping should remove stale projections
```

Comment:
```suggestion
    Scenario: SPEC Altering a workload mapping should remove stale projections
```
Please do not use duplicate scenario names - it causes conflicts in tools processing the results. The best practice here is that scenario names must be unique.

Reply:
Fixed in b370078b.